### PR TITLE
Tutorial for wrapping C++ operations for user in C++ and Python

### PR DESCRIPTION
**Pull request by arthurp**

### CODE REVIEW

File: docs/GraphOperationAPIs.md [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+The following elements should be documented, but you can document other things of course.
+
+1. Every constructor and static method constructor on the Plan.
+2. The operation function.
+3. Every member of the Statistics struct.
+
+### Complete Example: SSSP
+
+```c++
+#ifndef KATANA_LIBGALOIS_KATANA_ANALYTICS_SSSP_SSSP_H_
```
Comment (gurbinder533) [inline docs/GraphOperationAPIs.md]: Do we need the complete example here? Can we just have a pointer to the file?

**Review by gurbinder533 — APPROVED** [review]
LGTM.

File: docs/GraphOperationAPIs.md (hunk 1 above)
Comment (arthurp) [reply docs/GraphOperationAPIs.md]: Yea. You are probably right. I'll change it.

**Comment by arthurp** [comment]
@bowu, could I get a review on this. I had a change a few things since Gurbinder reviewed and I would like confirmation that this mostly makes sense based on your experiences. Obviously this document still needs work. But I want to get it in the repo anyway.

**Review by bowu — APPROVED** [review]
LGTM.

**Pull request merged**